wsd/ClientSession.hpp

```diff
@@ -50,12 +50,6 @@ class ClientSession final : public Session
                WAIT_DISCONNECT // closed and waiting for Kit's disconnected message
     );
 
-    enum ViewMode {
```

Comment:
this is useful, can we reintroduce that enum? :)

Reply:
Ah, yes, let me note this also. I will check these along with the JSON API.